pages/partials/question.ejs

```diff
@@ -98,7 +98,8 @@
               <% if (showSaveButton) { %><button class="btn btn-info question-save" name="__action" value="save">Save</button><% } %>
               <input type="hidden" name="__variant_id" value="<%= variant.id %>">
               <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
-              <% if (showNewVariantButton) { %><a href="<%= newVariantUrl %>" class="btn btn-primary">New variant</a><% } %>
+              <% if ((devMode || authz_data.has_instructor_edit) && urlPrefix.indexOf('/instructor') > 0) { %><a href="<%= urlPrefix %>/question/<%= question.id %>/edit" class="btn btn-success question-edit">Edit</a><% } %>
```

Comment:
We should add an explicit `instructor_mode` flag in `locals` if we don't have one already. This could be done in the `middlewares/authz*` functions.